### PR TITLE
Applied new main layout

### PR DESCRIPTION
**Pull request by cratz**
- Changes the default layout of header, footer, top menu, breadcrumb and projects menu
- cleaned up CSS
- Introduced SASS
- Adds icons for project menu entries
- New font for top menu and projects menu

### CODE REVIEW

**Comment by sschu** [comment]
I guess somebody has to adapt testunit tests checking the number of menu items and the like...

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -830,7 +830,7 @@ $(window).resize(function() {
 });
 
 $(window).bind('resizeEnd', function() {
-    jQuery("div#breadcrumb ul.breadcrumb").adjustBreadcrumbToWindowSize();
+    //jQuery("div#breadcrumb ul.breadcrumb").adjustBreadcrumbToWindowSize();
```
Comment (ulferts) [inline app/assets/javascripts/application.js]: if this method does nothing any more - please remove it.

**Comment by mfrister** [comment]
What about the idea of using non-readable Unicode code points for the icon font? In the SVG font I can see readable characters being used.

I wouldn't like merging this when it reduces accessibility.

**Comment by cratz** [comment]
Actually it doesn't reduce the accessibility without the non-readable Unicode codes since it will still work with all screen-readers.
A new version of the icon font is already in progress but not finished within the next few hours or days. We will move to non-readable Unicode codes after the icon font is updated.

**Comment by mfrister** [comment]
As just discussed, I don't want to merge this without having a license for the webfont. If we try to do this right, we shouldn't suddenly merge things without license.

I have not much of an idea about the CSS, but I'm pretty sure splitting things up in multiple files concerned with different parts of the layout is a good idea.

Apparently changing text to uppercase via CSS also affects Cucumber. To make the cukes not break with different CSS/Themes, I think we should make the matchers case-insensitive instead of adapting the cukes to look for uppercase text. Example: https://github.com/finnlabs/openproject-reporting/pull/39/files#diff-9c93ae044dade7e283e5dee7d3ba498dL143

**Comment by cratz** [comment]
fixed both

**Pull request merged**